test-network-function/diagnostic/suite.go

```diff
@@ -216,8 +216,8 @@ func testOcpVersion() {
 	test, err := tnf.NewTest(context.GetExpecter(), tester, []reel.Handler{tester}, context.GetErrorChannel())
 	gomega.Expect(err).To(gomega.BeNil())
 	testResult, err := test.Run()
```

Comment:
there is a RunAndValidate() that does what these 3 lines do

Reply:
ok sure. 
Yes, I believe the logging is needed. It is not captured otherwise.